build/templates/rep_caps.rst.mako

```diff
@@ -25,10 +25,15 @@ ${helper.get_rst_header_snippet('Repeated Capabilities', '=')}
 
     Repeated capabilities attributes use the indexing operator :python:`[]` to indicate the repeated capabilities.
     The parameter can be a string, list, tuple, or slice (range). Each element of those can be a string or
-    an integer. If it is a string, you can indicate a range using the same format as the driver: :python:`'0-2'` or
-    :python:`'0:2'`
+    an integer.
 
-    Some repeated capabilities use a prefix before the number and this is optional
+    ..
+        If it is a string, you can indicate a range using the same format as the driver: :python:`'0-2'` or
+        :python:`'0:2'`
+
+        Some repeated capabilities use a prefix before the number and this is optional.
+
+    The recommended way of accessing repeated capabilities is with an integer :python:`[0]` or range :python:`[0:2]`.
```

Comment:
I think I'm leaving out something important here.
:python;`[0,1,2]`

How do I refer to that? List? Tuple?

Reply:
It's actually a tuple. The '[]' is intended to be the indexing operator.